hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -83,7 +85,7 @@ def cache_dir(self) -> Path:
         return Path(f"/tmp/himl_cache/{self.__class__.__name__}-{self.encoder_type}/")
 
     def setup(self) -> None:
-        self.ssl_ckpt_run_id = ""
+        self.ssl_ckpt_run_id = self.ssl_checkpoint_run_id
```

Comment:
I feel this duplication could be confusing. How about consolidating it into a single parameter/attribute?